company/views.py

```diff
@@ -140,7 +145,7 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         context['report_list'] = Report.objects.only_open().filter(product__company=self.get_object())
-
+        context['capital_group'] = self.get_object().capital_group
```

Comment:
Czy jest to potrzebne? Wydaje mi się, że w szablonie można się do tego odwołać bezpośrednio.